src/AttestationData.ts

```diff
@@ -0,0 +1,408 @@
+// Email attestation dataStr type
+///////////////////////////////////////////////////
+export interface IBaseAttEmailData {
+  email?: string
+  start_date?: string
+  end_date?: string
+}
+export interface IBaseAttEmail extends IBaseAtt {
+  data: IBaseAttEmailData | Array<IBaseAttEmailData>
+}
+
+///////////////////////////////////////////////////
+// Name attestation dataStr type
+///////////////////////////////////////////////////
+export interface IBaseAttName extends IBaseAtt {
+  data: TPersonalName | Array<TPersonalName>
+}
+
+///////////////////////////////////////////////////
+// SSN/government ID # attestation dataStr type
+///////////////////////////////////////////////////
+export interface IBaseAttSSNData {
+  country_code: string
+  id_type: string
+  id: string | number
+}
+export interface IBaseAttSSN extends IBaseAtt {
+  data: IBaseAttSSNData | Array<IBaseAttSSNData>
+}
+
+///////////////////////////////////////////////////
+// Date of birth attestation dataStr type
+///////////////////////////////////////////////////
+export interface IBaseAttDOB extends IBaseAtt {
+  data: TDate
+}
+
+///////////////////////////////////////////////////
+// Account attestation dataStr type
+///////////////////////////////////////////////////
+export interface IBaseAttAccountData {
+  email?: string
+
+  name?: TPersonalName
+  start_date?: string
+  end_date?: string
+}
+export interface IBaseAttAccount extends IBaseAtt {
+  data: IBaseAttAccountData | Array<IBaseAttAccountData>
+}
+
+///////////////////////////////////////////////////
+// Sanction screen attestation dataStr type
+///////////////////////////////////////////////////
+export interface IBaseAttSanctionScreenData {
+  name: TPersonalName
+  birthday: TDate
+}
+export interface IBaseAttSanctionScreen extends IBaseAtt {
+  data: IBaseAttSanctionScreenData | Array<IBaseAttSanctionScreenData>
+}
+
+///////////////////////////////////////////////////
+// PEP screen attestation dataStr type
+///////////////////////////////////////////////////
+export interface IBaseAttPEPData {
+  date: TDate
+  name: TPersonalName
+  country: string
+
+  // Primarily modelled after KYC2020 responses, most fields left optional for flexibility
+  search_summary: {
+    hit_location?: string
+    hit_number?: number
+    list_name?: string
+    list_url?: string
+    record_id?: string
+    search_reference_id?: string
+    score?: string
+    hits?: Array<{
+      id?: string
+      hit_name?: string
+    }>
+    flag_type?: string
+    comment?: string
+  }
+}
+export interface IBaseAttPEP extends IBaseAtt {
+  data: IBaseAttPEPData | Array<IBaseAttPEPData>
+}
+
+///////////////////////////////////////////////////
+// ID document attestation dataStr type
+///////////////////////////////////////////////////
+export interface IBaseAttIDDocumentData {
+  date: TDate
+  name: TPersonalName
+  country: string
+
+  // Primarily modelled after KYC2020 responses, most fields left optional for flexibility
```

Comment:
All the fields in this interface seem to be required

Reply:
Not following